### PR TITLE
change join us email

### PR DESCRIPTION
**Pull request by isoworg**

### CODE REVIEW

File: source/join-the-team.html.haml [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-        We have graduated over #{stats.number_of_graduates} Makers who now work at Pivotal Labs, 8th Light, Thoughtworks, New Bamboo, BSkyB, FundingCircle, the Ministry of Justice, Global Personals, Mint Digital and many other great companies. We are profitable. Our mission is to teach as many people as possible to code well through our web development coding classes.
+        %span Since we started in early 2013 we grew from two people to a team of 20.
+        We've graduated more than #{stats.number_of_graduates} people, who now work at places like Pivotal Labs, 8th Light, Thoughtworks, New Bamboo, BSkyB, FundingCircle, the Ministry of Justice, Venntro, Mint Digital and many other great companies. We are profitable. Our mission is to teach as many people as possible to code well through our web development coding classes.
       %p To learn, read.
       %p To know, write.
       %p To master, coach.
   .button-row
-    %a{href: "mailto:careers@makersacademy.com", class: "button button--vertical"} Join us
+    %a{href: "mailto:jobs@makersacademy.com", class: "button button--vertical"} Join us
```
Comment (nikeshashar) [inline source/join-the-team.html.haml]: Does that email address work?

File: source/join-the-team.html.haml [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
+      %p Makers Academy is an exciting place to master technologies you've used for years. We're always open to applications.
 
 %section
   .article--centered
     %article
       %p
-        %span Since we started in early 2013 we grew from two people to a team of twenty.
-        We have graduated over #{stats.number_of_graduates} Makers who now work at Pivotal Labs, 8th Light, Thoughtworks, New Bamboo, BSkyB, FundingCircle, the Ministry of Justice, Global Personals, Mint Digital and many other great companies. We are profitable. Our mission is to teach as many people as possible to code well through our web development coding classes.
+        %span Since we started in early 2013 we grew from two people to a team of 20.
+        We've graduated more than #{stats.number_of_graduates} people, who now work at places like Pivotal Labs, 8th Light, Thoughtworks, New Bamboo, BSkyB, FundingCircle, the Ministry of Justice, Venntro, Mint Digital and many other great companies. We are profitable. Our mission is to teach as many people as possible to code well through our web development coding classes.
```
Comment (nikeshashar) [inline source/join-the-team.html.haml]: people -> Junior Developers

Comment (nikeshashar) [inline source/join-the-team.html.haml]: ThoughWorks

**Comment by danldb** [comment]
:shipit:

**Comment by isoworg** [comment]
@nikeshashar :shipit:

**Pull request merged**